__tests__/app/renderer/src/reducers/selectedIndex.js

```diff
@@ -8,6 +8,13 @@ describe('theme reducer', () => {
     expect(selectedIndex(state, action)).toEqual(0);
   });
 
+  it('should handle SELECT_ITEM', () => {
+    const state = 5;
+    const index = 1;
+    const action = { type: types.SELECT_ITEM, index: index };
```

Comment:
thanks to [property shorthand](http://es6-features.org/#PropertyShorthand), this can be the following

```js
const action = { type: types.SELECT_ITEM, index };
```

Can you correct that please. thanks